src/main/java/org/apache/commons/net/ftp/FTPSClient.java

```diff
@@ -204,7 +204,7 @@ public FTPSClient(final String protocol, final boolean isImplicit) {
 
     /**
      * Because there are so many connect() methods, the _connectAction_() method is provided as a means of performing some action immediately after establishing
-     * a connection, rather than reimplementing all of the connect() methods.
+     * a connection, rather than reimplementing all the connect() methods.
      *
      * @throws IOException If it throw by _connectAction_.
```

Comment:
Still not correct.

Reply:
That's what was there before. 
Any suggestion how that should be formulated?

Reply:
> That's what was there before. Any suggestion how that should be formulated?

Use a JDK method as a template

Reply:
I've changed it according to the style found/used in `FTPSClient`.
To be found in next commit together with changes proposed by @sebbASF.